### ISSUE TITLE
training from scratch

### ISSUE THREAD

**Issue by liminn** [poppinace/indexnet_matting#13]
Thanks for great work~
I am trying to training your model from scratch(only use pretrained mobilenet weights).
I encountered two problems:
<img width="654" alt="屏幕快照 2019-11-27 下午8 57 15" src="https://user-images.githubusercontent.com/30917208/69725570-6efc2000-1159-11ea-990f-5c56a953515d.png">
- The first problem is shown by the red arrow: the alpha value of the  unkonwn region is not large enough.
- The second problem is shown by the blue arrow: outside the unknown region, there are always white scattered dots.

For the first problem, I think my training epochs is not enough (just trained to the 6th epoch). 
For the second problem, I am very confused and have no ideas.
Do you have any suggestions on these two problems?

**Comment by poppinace** [comment]
Hi @liminn,

In my opinion, I think your ground truth alpha mattes are not generately correctly. Please check your code making sure the alpha is within the range [0, 1].

Hao

**Comment by liminn** [comment]
@poppinace Thanks, I will check the alpha labels.

**Comment by facetohard** [comment]
Hello @liminn @poppinace , I am trying to train indexnet from scratch.
